docs/reST/ref/surface.rst

```diff
@@ -134,8 +134,8 @@
 
       | :sl:`draw many images onto another`
       | :sg:`blits(blit_sequence=(source, dest), ...), doreturn=1) -> [Rect, ...] or None`
```

Comment:
This one still needs another parenthesis.

Reply:
Somehow I didn't even notice that  one! Thanks.